Assets/HoloToolkit/Build/Editor/BuildDeployPortal.cs

```diff
@@ -0,0 +1,438 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+
+using UnityEngine;
+using System.Collections;
+using System.Net;
+using System;
+using System.IO;
+using System.Collections.Generic;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Function used to communicate with the device through the REST API
+    /// </summary>
+    public class BuildDeployPortal
+    {
+        // Consts
+        public const float kTimeOut = 6.0f;
```

Comment:
We recommend and follow general [C# coding conventions](https://msdn.microsoft.com/en-us/library/ff926074.aspx). Also match the style in files for sure.

Personally, I can do without the 'k' prefix :)